Added "PhantomMode"

bilgeonceken opened the pull request:
When collisions occur the user cannot read collided lectures' names. To solve the problem PhantomMode button makes every lecture transparent except one and lets the user iterate through the lectures one by one.

bilgeonceken committed:
Added "PhantomMode"
Phantom mode makes every lecture transparent except one and lets you
cycle through lectures.

libduck2 commented:
thx a few issues

- in tetick I have denied javascript's overall style and avoided camelcase where possible- better keep style nonsense but consistent
- we draw data from DOM in lectList, no need for that. blocks[] defined at the very start have that information I'll elaborate as code quickly
- PhantomMode is again inconsistent with UI's overall style, that button can have a name like "see phantoms" or "flashlight" or "phantom mode" or something. open for suggestions
- as an UI precaution prevp and nextp buttons can be `display:none` at first and pop up when the user clicks on "phantom mode". this way it won't be confused with schedule prev/next
- as another UI thing we can tie up/down arrows to prevp/nextp

Commit:
don't draw data from dom

Commit:
more style nitpicks

now it looks like my code :Dd

bilgeonceken committed:
"PhantomMode" is now "Flashlight"
Also Next and Previous buttons now appear when "Flashlight" button is clicked, disappear when a new table has drawn.

bilgeonceken committed:
"Flashlight" is now a toggle button
Clicking second time turns it off. Also added an extra if condition to
prevent it from showing "next" and "previous" buttons if there are no
lecture block.

Commit:
pls don't basbursen the code

Commit:
further de-doldur.xyz ification

Commit:
i hate conditionals

Commit:
fix layout

Commit:
intuitive iteration

Commit:
remove onclicks

Commit:
wtf halfcommit

libduck2 commented:
take a look this is now mergeable I think
